src/solvers/floatbv/float_bv.cpp

```diff
@@ -810,9 +812,9 @@ exprt float_bvt::relation(
       not_exprt(nan));
   }
   else
-    assert(0);
+    UNREACHABLE;
```

Comment:
Maybe this `else` branch should just be removed? It's redundant with the code afterwards.